feat(changelog): transactional-email-added-addition-of-smtp-host 2023-04-27 (#1601)
scaleway/docs-content: changelog/april2023/2023-04-27-transactional-email-added-addition-of-smtp-host.mdx

---

> title: Addition of SMTP host and ports in tem_domain Terraform resource

Reviewer: ```suggestion
title: Addition of an SMTP host and ports in Terraform
```

status: added
author:

>     fullname: 'Join the #[xxx] channel on Slack.'

Reviewer: ```suggestion
    fullname: 'Join the #transactional-email channel on Slack.'
```

    url: 'https://slack.scaleway.com'
date: 2023-04-27
category: managed-services
product: transactional-email
---

> Addition of SMTP host and ports in tem_domain Terraform resource.

Reviewer: ```suggestion
We have [added an SMTP host and ports](https://registry.terraform.io/providers/scaleway/scaleway/latest/docs/data-sources/tem_domain) in the `tem_domain` Terraform resource. 
```

> https://registry.terraform.io/providers/scaleway/scaleway/latest/docs/data-sources/tem_domain

Reviewer: ```suggestion
```
